addons/addon-base-raas/packages/base-raas-services/lib/study/__tests__/study-service.test.js

```diff
@@ -251,6 +251,50 @@ describe('studyService', () => {
       );
     });
 
+    it('should fail to update resource list of non-Open Data study', async () => {
+      // BUILD
+      const dataIpt = {
+        id: 'newOpenStudy',
+        category: 'Organization',
+        resources: [{ arn: 'arn:aws:s3:::someRandomStudyArn' }],
+      };
+      service.audit = jest.fn();
+
+      // OPERATE
+      try {
+        await service.update(
+          { principal: { userRole: 'researcher' }, principalIdentifier: { uid: 'someRandomUserUid' } },
+          dataIpt,
+        );
+        expect.hasAssertions();
```

Comment:
Do we expect atleast one assertion for all tests in this file? If yes, we can put ` expect.hasAssertions();` in the `beforeEach`.

Reply:
Hmm, not sure if we should. Most of the tests are built are on a case by case basis. Some require the assertion check since they depend on async calls, while some don't.